lib/matplotlib/mpl-data/matplotlibrc

```diff
@@ -425,8 +425,9 @@
 #axes.autolimit_mode: data  # If "data", use axes.xmargin and axes.ymargin as is.
                             # If "round_numbers", after application of margins, axis
                             # limits are further expanded to the nearest "round" number.
-#polaraxes.grid: True  # display grid on polar axes
-#axes3d.grid:    True  # display grid on 3D axes
+#polaraxes.grid:    True   # display grid on polar axes
+#axes3d.grid:       True   # display grid on 3D axes
+#axes3d.automargin: False  # automatically add margin when manually setting 3D axis limits
```

Comment:
I do not fully get if this or "classic" should be True/False, but shouldn't they be different?

Reply:
I'm not sure I follow?

Reply:
Maybe I am not interpreting "classic" correctly, but I was just thinking that "classic" possibly should have `automargin=True`, to preserve the "classic" style?

Reply:
Is 'classic' the default? I'm struggling to find where else the default style is stored. Per the comment up above that we want to "make the new behavior default without deprecation", I agree it makes sense if this is a style the user has to apply, but not if it actually is the default.

Reply:
I *think* classic is pre-2.0 (as there is an mpl20 style). But will see if I can dig it up.

Reply:
Now, I've thought a bit about it and I think it makes sense to set it to True in the classic style sheet. In that way, using that style sheet will have the pre-2.0 look. It only affects a few of the tests (the ones that to not use `style='mpl20'`). 

(If I understand things.)

Reply:
Yes, the "classic" style sheet is from the 2.0 release which we provided so that users who wanted to go back _can_ and we use it internally (plus some extra settings) in our tests to avoid having to re-generate a lot of images.